rocks.noise.QSpeakers.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: rocks.noise.QSpeakers
```

Comment:
how do I know you control https://noise.rocks/ ? Can you link the github somewhere?